### PR TITLE
Refresher

### PR DESCRIPTION
**Pull request by yohanology**
*Add Pull to refresh feature

### CODE REVIEW

File: www/js/app.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ angular.module('starter', ['ionic', 'starter.controllers', 'ng-token-auth'])
     }
     if (window.StatusBar) {
 
-      StatusBar.styleDefault();
+      StatusBar.style(1);
```
Comment (h8rry) [inline www/js/app.js:13]: good

File: www/js/controllers.js [hunk 2]
```diff
@@ -215,15 +220,19 @@ angular.module('starter.controllers', [])
...
   var user_id = $scope.currentUser.id;
 
-  // Get list of forms for current user in submissions
-  $http.get(apiUrl + "/users/" + user_id + "/forms").success(
-    function(resp){
-      console.log("list forms",resp);
-      $scope.forms = resp;
-
-  }).error(function(resp){
+  $scope.doRefresh = function(){
```
Comment (h8rry) [inline www/js/controllers.js]: Duplicated. services?

File: www/js/controllers.js [hunk 3]
```diff
@@ -237,15 +246,21 @@ angular.module('starter.controllers', [])
     console.log(resp)
   });
 
-  // Get list of submissions for 1 form
-  $http.get(apiUrl + "/forms/" + $stateParams.id + "/submissions").success(
-    function(resp){
-      console.log("list submissions",resp);
-      $scope.submissions = resp;
+  $scope.doRefresh = function(){
```
Comment (h8rry) [inline www/js/controllers.js]: Dup

File: www/js/controllers.js [hunk 4]
```diff
@@ -56,14 +56,19 @@ angular.module('starter.controllers', [])
   
   var user_id = $scope.currentUser.id;
 
-  // Get list of forms for current user
-  $http.get(apiUrl + "/users/" + user_id + "/forms").success(
-    function(resp){
-      console.log(resp);
-      $scope.forms = resp;
+  $scope.doRefresh = function(){
```
Comment (h8rry) [inline www/js/controllers.js]: Dup

**Comment by h8rry** [comment]
@yohanology please review

**Comment by yohanology** [comment]
I tried to put it into services but i can't make it work because of dependencies/injections errors. I just renamed  some functions because some of them don't do exactly the same things (refresh submissions or refresh forms).

**Pull request merged**